tox.ini

```diff
@@ -21,7 +21,7 @@ deps =
     pytest==3.10.1
     requests==2.7.0
 
-[testenv:py{36,37,38,39}-sa{13,14}]
+[testenv:py39-sa14]
```

Comment:
To reduce test runtime, I've changed the py39 test to only include sa14, and the 35 test includes only sa13. Tests can complete now in under 15 minutes.